api/lib/domain/models/CertificationAssessment.js

```diff
@@ -45,6 +45,12 @@ class CertificationAssessment {
     return _.find(this.certificationChallenges, { challengeId }) || null;
   }
 
+  neutralizeChallengeByRecId(recId) {
+    const challengeToBeNeutralized = this.certificationChallenges.find((challenge) => challenge.challengeId === recId);
+    if (challengeToBeNeutralized) { // TODO : throw if not found ?
```

Comment:
oui, moi je throwerai 404 Not Found je pense en mode "L'épreuve que vous cherchez à neutraliser n'a pas été posée pour le test de certification"